<launch>
	
    <group ns="armar6">
        <node pkg="giskardpy" type="giskard.py" name="giskard" output="screen">
            <param name="config" value="Armar6Unreal"/>
        </node>
    
        <node pkg="giskardpy" type="interactive_marker.py" name="giskard_interactive_marker" output="screen">
            <rosparam param="enable_self_collision">False</rosparam>
            <rosparam param="interactive_marker_chains">
                - [odom, Hand L Palm]
                - [odom, Hand R Palm]
                - [odom, Roboception]
            </rosparam>
        </node>
    </group>

</launch>